languages/go/tests/unit_test.go

```diff
@@ -21,30 +21,41 @@ func TestNewOso(t *testing.T) {
 	}
 }
 
-func TestLoadFile(t *testing.T) {
+func TestLoadFiles(t *testing.T) {
 	var o oso.Oso
 	var err error
 	if o, err = oso.NewOso(); err != nil {
 		t.Fatalf("Failed to set up Oso: %v", err)
 	}
 
-	if err = o.LoadFile("test.polar"); err != nil {
-		t.Error(err.Error())
-	}
-
-	if err = o.LoadFile("test.polar"); err == nil {
+	if err = o.LoadFiles([]string{"test.polar", "test.polar"}); err == nil {
 		t.Error("Failed to error on loading duplicate file")
 	}
 
-	if err = o.LoadFile("test.txt"); err == nil {
+	if err = o.LoadFiles([]string{"test.txt"}); err == nil {
 		t.Error("Failed to error on loading non-polar file (.txt)")
 	}
 
-	if err = o.LoadFile("fake.polar"); err == nil {
+	if err = o.LoadFiles([]string{"fake.polar"}); err == nil {
 		t.Error("Failed to error on loading non-existent file")
 	}
 }
 
+// test_load_multiple_files_same_name_different_path
+func TestLoadMultipleFilesSameNameDifferentPath(t *testing.T) {
+	t.Skip("Fixing test is outside scope of current task.")
```

Comment:
Did you identify what was going on here?

Reply:
Figured it out in 74e0c3e.

Turns out this was a bug in `LoadFiles` that prevented loading >1 files 😬